refactor(docs): split monolithic api reference into multiple files (#722)
DisnakeDev/disnake: docs/api/widgets.rst

.. SPDX-License-Identifier: MIT

.. currentmodule:: disnake

Widgets
=======

> This section documents everything related to widgets - Discord feature which allows to integrate a guild's "widget" - dynamically
> generated image which shows basic info about your server and invites to join it without having to authenticate - into any place supporting HTML.

Reviewer: ```suggestion
This section documents everything related to widgets — dynamically
generated elements which show basic info about your server and invites to join it easily — into any place supporting HTML.
```
